src/test/kotlin/test/assertk/assertions/PathSpec.kt

```diff
@@ -0,0 +1,104 @@
+package test.assertk.assertions
+
+import assertk.assert
+import assertk.assertions.*
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.jetbrains.spek.api.Spek
+import org.jetbrains.spek.api.dsl.given
+import org.jetbrains.spek.api.dsl.it
+import java.nio.file.Paths
+
+class PathSpec: Spek({
+
+    given("a regular file") {
+        val regularFile = Paths.get("LICENSE")
+
+        it("is a regular file") {
+            assert( regularFile ).isRegularFile()
+        }
+
+        it("is not a folder") {
+            assertThatThrownBy {
+                assert( regularFile ).isDirectory()
+            }.hasMessage("expected <LICENSE> to be a directory, but it is not")
+        }
+
+//        it("is not an executable") {
```

Comment:
Why is this commented out?

Reply:
Because under windows files or folders are "executable" (according to Files.isEecutable()) so this test doesn't work like you would expect it

Reply:
Oh weird, yeah if this method is platform specific it might be hard to test.